tests/unit_tests/gce/test_cloud.py

```diff
@@ -156,3 +156,34 @@ def test_daily_image_returns_latest_from_query(  # noqa: D102
             mock.call("jammy", "project-name", "name-filter", "x86_64")
         ]
         assert image == "projects/project-name/global/images/4"
+
+    @pytest.mark.parametrize(
+        ["release", "image_type", "expected_name_filter"],
+        [
+            pytest.param(
+                "jammy",
+                ImageType.GENERIC,
+                "daily-ubuntu-2204-jammy-*",
+            ),
+            pytest.param(
+                "noble",
+                ImageType.MINIMAL,
+                "daily-ubuntu-minimal-2404-noble-*",
+            ),
+            pytest.param(
+                "focal",
+                ImageType.PRO,
+                "ubuntu-pro-2004-focal-*",
+            ),
+            pytest.param(
+                "focal",
+                ImageType.PRO_FIPS,
+                "ubuntu-pro-fips-2004-focal-*",
+            ),
+        ],
+    )
+    def test_get_name_filter(self, release, image_type, expected_name_filter):
+        gce = FakeGCE(tag="tag")
```

Comment:
Same, fake tag=tag param unnecessary. not a blocker to landing. Resolve or not and merge as you see fit.

Reply:
alright removed all references to tags for these fake classes since its a noop.